authors/apps/authentication/renderers.py

```diff
@@ -18,7 +18,6 @@ def render(self, data, media_type=None, renderer_context=None):
             # rendering errors.
             return super(UserJSONRenderer, self).render(data)
```

Comment:
While I believe this is to support backwards compatibility with python2.7, Django no longer supports python2.7, so I don't think it is necessary here. What do you think?

Reply:
You are right, let me update that to support only Python3